app/assets/stylesheets/application.css.scss

```diff
@@ -98,7 +98,7 @@ article {
   border-radius: 5px 5px 5px 5px;
   background-color: rgba(247, 247, 247, 0.5);
   padding: 5px;
-  margin: 50px 80px 10px 50px;
+  margin: 50px 50px 10px 50px;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `50px 50px 10px` instead of `50px 50px 10px 50px`